### PR TITLE
Fix required status check

### PR DESCRIPTION
**Pull request by trask**
This fixes the problem that I was able to merge #1544 even though one of the tests was failing.

### CODE REVIEW

File: .github/workflows/build.yml
Comment (breedx-splk) [inline .github/workflows/build.yml:156]: I'm not following how this helps. Isn't `&& always()` basically `&& true` which is always true?

Comment (trask) [reply .github/workflows/build.yml:156]: one would think...

check out https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always

and you can see the difference below, `required-status-check` is run and fails (and so should block merging even after someone approves this PR), whereas in #1542 for example, `required-status-check` is skipped (and therefore didn't block merging)

**Comment by trask** [comment]
I'm going to rebase now that #1546 has been merged, which should fix the integration tests and make `required-status-check` pass (allowing this PR to be merged)

**Review by breedx-splk — APPROVED** [review]

**Pull request merged**
